### ISSUE TITLE
2.模型的初步建立，线性层，卷积层等

### ISSUE THREAD

**Issue by renxianxu** [renxianxu/MathStudy-Machine-Learning#3]
import torch.nn as nn  # 导入神经网络模块
import torch.nn.functional as F  # 导入常用的激活函数、池化等操作

# 1. 定义Net类，继承自nn.Module
class Net(nn.Module):
    
    # 2. 初始化函数：定义神经网络的各层
    def __init__(self):
        super(Net, self).__init__()  # 调用父类nn.Module的初始化函数，初始化网络层
        # 定义第一个卷积层：输入1通道，输出20通道，卷积核大小为5x5，步幅为1
        self.conv1 = nn.Conv2d(1, 20, 5, 1)  
        # 定义第二个卷积层：输入20通道，输出50通道，卷积核大小为5x5，步幅为1
        self.conv2 = nn.Conv2d(20, 50, 5, 1)  
        # 定义第一个全连接层：输入大小为4*4*50，输出500
        self.fc1 = nn.Linear(4*4*50, 500)  
        # 定义第二个全连接层：输入500，输出10（即10个类别）
        self.fc2 = nn.Linear(500, 10)  

    # 3. 定义前向传播函数：定义数据流如何通过网络的各层
    def forward(self, x):
        # 通过第一个卷积层 + ReLU激活 + 最大池化
        x = F.relu(self.conv1(x))  # 使用ReLU激活函数
        x = F.max_pool2d(x, 2, 2)  # 使用2x2最大池化
        
        # 通过第二个卷积层 + ReLU激活 + 最大池化
        x = F.relu(self.conv2(x))  # 使用ReLU激活函数
        x = F.max_pool2d(x, 2, 2)  # 使用2x2最大池化
        
        # 将卷积层的输出展平为一维，准备输入到全连接层
        x = x.view(-1, 4*4*50)  # 展平操作，-1表示自动推算这一维的大小
        
        # 通过第一个全连接层 + ReLU激活
        x = F.relu(self.fc1(x))  # 使用ReLU激活函数
        
        # 通过第二个全连接层
        x = self.fc2(x)  # 第二个全连接层，输出10个类别
        
        # 返回log_softmax结果，用于多分类任务
        return F.log_softmax(x, dim=1)  # 输出log_softmax结果，适用于分类任务

# 4. 创建模型实例并打印模型结构
model = Net()  # 创建Net类的实例
print(model)  # 打印模型结构，显示每一层的详细信息

output：Net(
  (conv1): Conv2d(1, 20, kernel_size=(5, 5), stride=(1, 1))
  (conv2): Conv2d(20, 50, kernel_size=(5, 5), stride=(1, 1))
  (fc1): Linear(in_features=800, out_features=500, bias=True)
  (fc2): Linear(in_features=500, out_features=10, bias=True)
)
